caikit/runtime/servicers/global_predict_servicer.py

```diff
@@ -250,7 +253,7 @@ def predict_model(
                 inference request
         """
 
-        with self._handle_predict_exceptions(model_id, request_name):
+        with (self._handle_predict_exceptions(model_id, request_name)):
```

Comment:
Curious why the extra parens are needed?

Reply:
idk man I just ran `black` lol

Reply:
aaaand if I remove + rerun, they don't reappear. Odd